cds-snc/report-a-cybercrime
# add newline to cert if needed

Fixes #1357 

# Description

Add a newline to the cert (before the `-----END CERTIFICATE-----`) if needed. Without it openssl fails.

To verify: 
1) in your .env, set `LDAP_UID` to Kevin's UID
2) in master run `npm run build && node server.js`
3) run through the app, after submitting the terminal line will say "Encrypted Mail: Message encrypted"

Note that on master step3 results in the server crashing due to an openssl error.

# Checklist:
